README.md

```diff
@@ -1,3470 +1,188 @@
+clientConfig.networkConfig.addresses.push('10.90.0.2:5701');
+clientConfig.networkConfig.addresses.push('10.90.0.3:5701');
 
- By default, Hazelcast Node.js client uses a default logger which logs to the `stdout` with the `INFO` log level. You can change the log level using the `'hazelcast.logging.level'` property of the `ClientConfig.properties`.
- 
-Below is an example of the logging configuration with the `OFF` log level which disables logging.
- 
-```javascript
-cfg.properties['hazelcast.logging.level'] = LogLevel.OFF;
+// Initialize the client with the given configuration
+Client.newHazelcastClient(clientConfig)
+  .then(client => {
+    console.log('Connected to cluster');
+    return client.shutdown();
+  });
 ```
- 
- You can also implement a custom logger depending on your needs. Your custom logger must have `log`, `error`, `warn`, `info`, `debug`, `trace` methods. After implementing it, you can use your custom logger using the `customLogger` property of `ClientConfig`
-  
-See the following for a custom logger example.
- 
-```javascript
-var winstonAdapter = {
-    logger: new (winston.Logger)({
-        transports: [
-            new (winston.transports.Console)()
-        ]
-    }),
-
-    levels: [
-        'error',
-        'warn',
-        'info',
-        'debug',
-        'silly'
-    ],
-
-    log: function (level, objectName, message, furtherInfo) {
-        this.logger.log(this.levels[level], objectName + ': ' + message, furtherInfo);
-    },
-
-    error: function (objectName, message, furtherInfo) {
-        this.log(LogLevel.ERROR, objectName, message, furtherInfo);
-    },
-
-    warn: function (objectName, message, furtherInfo) {
-        this.log(LogLevel.WARN, objectName, message, furtherInfo);
-    },
 
-    info: function (objectName, message, furtherInfo) {
-        this.log(LogLevel.INFO, objectName, message, furtherInfo);
-    },
+You can also configure the client
+[declaratively](https://github.com/hazelcast/hazelcast-nodejs-client#312-declarative-configuration-json) using a JSON file.
 
-    debug: function (objectName, message, furtherInfo) {
-        this.log(LogLevel.DEBUG, objectName, message, furtherInfo);
-    },
+## Features
 
-    trace: function (objectName, message, furtherInfo) {
-        this.log(LogLevel.TRACE, objectName, message, furtherInfo);
-    }
+* Distributed, partitioned and queryable in-memory key-value store implementation, called **Map**
+* Eventually consistent cache implementation to store a subset of the Map data locally in the memory of the client, called **Near Cache**
+* Additional data structures and simple messaging constructs such as **Set**, **MultiMap**, **Queue**, **Topic**
+* Cluster-wide unique ID generator, called **FlakeIdGenerator**
+* Distributed, CRDT based counter, called **PNCounter**
+* Primitives for distributed computing such as **Lock**, **Semaphore**, **Atomic Long**
+* Integration with [Hazelcast.cloud](https://cloud.hazelcast.com/)
```

Comment:
Don't we call it `Hazelcast Cloud` (not `Hazelcast.cloud`)?

Reply:
We were calling it Hazelcast.cloud while implementing the discovery feature but it is called as Hazelcast Cloud in other places now. Updated it